frontend/src/list-default/list-default.js

```diff
@@ -1,10 +1,27 @@
 'use strict';
 
 const template = require('./list-default.html');
+const appendCSS = require('../appendCSS');
+
+appendCSS(require('./list-default.css'));
 
 module.exports = app => app.component('list-default', {
   template: template,
   props: ['value'],
+  methods: {
+    copyText(value) {
+      console.log('I have been clicked!')
+      const storage = document.createElement('textarea');
+      storage.value = value;
+      const elem = document.querySelector('#value-field');
```

Comment:
Woops, I missed this. Do **not** rely on id, this will only work for the first list-default component on the page.

Use a Vue ref instead.